subjects/revconcatalternate/README.md

```diff
@@ -0,0 +1,45 @@
+## Reverse-Concat-Alternate
+
+
+### Instructions
+
+- Write a function that receives two slices of int as arguments and returns a new one with the result of alternating the values of each in reverse order.
```

Comment:
It's better if remove `-` form first line

Reply:
fixed